test/functional/test_runner.py

```diff
@@ -248,6 +249,7 @@ def run_tests(test_list, src_dir, build_dir, exeext, jobs=1, enable_coverage=Fal
     results = BOLD[1] + "%s | %s | %s\n\n" % ("TEST".ljust(max_len_name), "PASSED", "DURATION") + BOLD[0]
     for _ in range(len(test_list)):
         (name, stdout, stderr, passed, duration) = job_queue.get_next()
+        name = " ".join(name)
```

Comment:
Ah, of course.

Mind keeping them separate so that's more obvious? e.g. rename to test and print test[0] + test[1:] ?